testutils/ocispec.go

```diff
@@ -153,6 +153,33 @@ func AreSyscallsBlockedBySeccomp(seccompProfile specs.LinuxSeccomp, syscallNames
 	return true
 }
 
+// isSyscallWithArgsAllowedBySeccomp checks that the provided syscall and args are whitelisted by the seccomp profile
+func isSyscallWithArgsAllowedBySeccomp(seccompProfile specs.LinuxSeccomp, syscallName types.Syscall, syscallArgs []specs.LinuxSeccompArg) bool {
+	return !isSyscallWithArgsBlockedBySeccomp(seccompProfile, syscallName, syscallArgs)
+}
+
+// AreSeccompSyscallsWithArgsAllowed checks that the provided list of syscalls and args are whitelisted by the seccomp profile
+func AreSeccompSyscallsWithArgsAllowed(seccompProfile specs.LinuxSeccomp, syscallsWithArgs map[types.Syscall][]specs.LinuxSeccompArg) bool {
+	for syscallName, syscallArgs := range syscallsWithArgs {
+		if !isSyscallWithArgsAllowedBySeccomp(seccompProfile, syscallName, syscallArgs) {
```

Comment:
Can we test this?

Reply:
you mean `isSyscallWithArgsAllowedBySeccomp` or `AreSeccompSyscallsWithArgsAllowed` ?

Reply:
The case where the inner conditional `if !isSyscallWithArgsAllowedBySeccomp(seccompProfile, syscallName, syscallArgs)` is true and the return value for `AreSeccompSyscallsWithArgsAllowed` is `false`